File: catalog/admin/cat/ordering.php
<?php
defined('AUTH') or die('Restricted access');

if(isset($_POST['cancel']))
	header('location: /admin/com/catalog');

include_once $_SERVER['DOCUMENT_ROOT'].'/components/catalog/admin/classes/AdminCatalogCat.php';

$catalog = new AdminCatalogCat();

$act = $SITE->p[4];
$id = $SITE->p[5];

$catalog->ordering($act, $id);

header('location: /admin/com/catalog');

File: dan/system/catalog/cat/ordering.py
import sys
sys.path.append('system/catalog/classes')
from system.catalog.classes.Catalog import Catalog


def ordering(SITE):
    print('FUNCTION -> system-> calalog -> cat -> ordering')
    type = SITE.p[2]
    id = SITE.p[3]

    CATALOG = Catalog(SITE)
    catalog = CATALOG.ordering(type, id)

    if 'cancel' in SITE.post:
        return {'redirect': '/system/catalog/cat'}
    return {'redirect': '/system/catalog/cat'}
